sycl/include/sycl/detail/accessor_impl.hpp

```diff
@@ -79,11 +80,29 @@ class __SYCL_EXPORT AccessorImplHost {
   AccessorImplHost(id<3> Offset, range<3> AccessRange, range<3> MemoryRange,
                    access::mode AccessMode, detail::SYCLMemObjI *SYCLMemObject,
                    int Dims, int ElemSize, int OffsetInBytes = 0,
-                   bool IsSubBuffer = false, bool IsESIMDAcc = false)
+                   bool IsSubBuffer = false, bool IsESIMDAcc = false,
+                   const property_list &PropertyList = {})
       : MOffset(Offset), MAccessRange(AccessRange), MMemoryRange(MemoryRange),
         MAccessMode(AccessMode), MSYCLMemObj(SYCLMemObject), MDims(Dims),
         MElemSize(ElemSize), MOffsetInBytes(OffsetInBytes),
-        MIsSubBuffer(IsSubBuffer), MIsESIMDAcc(IsESIMDAcc) {}
+        MIsSubBuffer(IsSubBuffer), MIsESIMDAcc(IsESIMDAcc),
+        MPropertyList(PropertyList) {}
+  template <typename... Props>
```

Comment:
```suggestion

  template <typename... Props>
```
Nit, but it makes it easier to read. 😄

Reply:
Remove this section since it is redundant now